src/utils/body.ts

```diff
@@ -60,23 +75,57 @@ function convertFormDataToBodyData<T extends BodyData = BodyData>(
     const shouldParseAllValues = options.all || key.endsWith('[]')
 
     if (!shouldParseAllValues) {
-      form[key] = value
+      if (options.dot && key.includes('.')) {
+        setNestedValue(form, key.split('.'), value)
```

Comment:
Maybe don't split the keys here to make the API look similar to `handleParsingAllValues`. You can do it inside the func too.

Reply:
Sure, will try same pattern